build/data/services/service_sql.txt

```diff
@@ -0,0 +1,23 @@
+// SQL for building the curl command to insert 1000 services into elasticsearch
```

Comment:
Can we move this file to `data` directory instead of the `build/data`?

Reply:
Done